build-data/paper.at

```diff
@@ -236,3 +236,7 @@ public-f com.mojang.brigadier.tree.CommandNode requirement
 # Block Enderpearl Travel Exploit
 public net.minecraft.world.entity.projectile.Projectile cachedOwner
 public net.minecraft.world.entity.projectile.Projectile ownerUUID
+
+# Add methods for lightning target lookups
```

Comment:
Use the name of the server patch here, ie `Add methods to find targets for lightning strikes`